helix-core/src/main/java/org/apache/helix/manager/zk/ParticipantManager.java

```diff
@@ -501,6 +515,10 @@ private void setupMsgHandler() throws Exception {
     _messagingService.onConnected();
   }
 
+  private void setupInstanceConfigChangeHandler() throws Exception {
+
+  }
```

Comment:
What does this empty method do? Seems like not used anywhere?

Reply:
I forgot to remove this method, thanks for catching it.